canonical/ubuntu.com
# WD-8569 - refresh service terms

## Done

Update service terms page according to the copy doc

Note: after merge update the date of update at the bottom of the copy doc.

## QA
- [demo link](https://ubuntu-com-13578.demos.haus/core/stories)
- [copy doc](https://docs.google.com/document/d/1aeqs6k-jgTf8xu3bOcloYpRgvIy7DuGjwyk7TLeTcqw/edit)
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correctly updated according to copy doc

## Issue / Card
[WD-8569](https://warthogs.atlassian.net/browse/WD-8569)

Fixes #

## Screenshots
